File: app/src/main/java/com/hypnoweb/hypnowebapp/di/modules/FragmentBuilderModule.java
package com.hypnoweb.hypnowebapp.di.modules;


import com.hypnoweb.hypnowebapp.splashscreen.SplashScreenFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class FragmentBuilderModule {
    @ContributesAndroidInjector
    abstract SplashScreenFragment contributeSplashScreenFragmentInjector();

}

File: app/src/main/java/com/hypnoweb/hypnowebapp/di/modules/FragmentBuilderModule.kt
package com.hypnoweb.hypnowebapp.di.modules

import com.hypnoweb.hypnowebapp.ui.splashscreen.SplashScreenFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector


@Module
abstract class FragmentBuilderModule {
    @ContributesAndroidInjector
    abstract fun contributeSplashScreenFragmentInjector(): SplashScreenFragment

}
